src/snowflake/snowpark/functions.py

```diff
@@ -514,14 +555,36 @@ def count_distinct(*cols: ColumnOrName) -> Column:
 
 
 def covar_pop(column1: ColumnOrName, column2: ColumnOrName) -> Column:
-    """Returns the population covariance for non-null pairs in a group."""
+    """Returns the population covariance for non-null pairs in a group.
+
+    Example:
+        >>> df = session.create_dataframe([[1, 2], [1, 2], [4, 5], [2, 3], [3, None], [4, None], [6,4]], schema=["a", "b"])
+        >>> df.select(covar_pop(col("a"), col("b")).alias("result")).show()
+        ----------------------
+        |"RESULT"            |
+        ----------------------
+        |1.8400000000000005  |
```

Comment:
Can we do a cast to only return `1.84`? I'm not sure how consistent the result would be for the 15th digit after decimal point.